StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -65,6 +69,9 @@ final class PaymentSheetLoader {
                             intent: intent
                         )
                     }
+
+                let loadingEndTime = CFAbsoluteTimeGetCurrent() - loadingStartTime
```

Comment:
nit: This should be called duration, not loadingEndTime.  

ooc, why `CFAbsoluteTimeGetCurrent` instead of `Date`?

Reply:
When looking for referencing material on how to measure execution speed in Swift the most common suggestion is `CFAbsoluteTimeGetCurrent` from what I've seen. [1,](https://www.hackingwithswift.com/example-code/system/measuring-execution-speed-using-cfabsolutetimegetcurrent) [2](https://stackoverflow.com/questions/25006235/how-to-benchmark-swift-code-execution), but there is a bit of [debate](https://news.ycombinator.com/item?id=22824496).

When looking at [Date.swift](https://github.com/apple/swift-corelibs-foundation/blob/main/Sources/Foundation/Date.swift#L27C16-L27C40) it's just using `CFAbsoluteTimeGetCurrent` under the hood. `Date` is probably a bit more idiomatic so I'll swap here as it's equivalent.